servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -627,12 +629,17 @@ protected Stream<KeyEntry<Content.Type>> filterEntries(
           // currently this is just a workaround where we put EntriesResponse.Entry into a hash
           // structure.
           // Eventually we should just be able to do "script.execute(Boolean.class, entry)"
+          ContentKey contentKey = fromKey(entry.getKey());
           Map<String, Object> arguments =
               ImmutableMap.of(
                   VAR_ENTRY,
                   ImmutableMap.of(
+                      VAR_CONTENT_KEY,
```

Comment:
Maybe expose these attributes using the same "structure" that `OperationForCel` already has, with different properties for the full string plus individual elements.

Reply:
Good point. thx! I've refactored the entry filter script invocation to use CEL wrappers.